myapp/src/components/Authentication/ProfileForm/ProfileFormElements.js

```diff
@@ -1,6 +1,6 @@
 import styled from 'styled-components';
 import { NavLink as Link } from 'react-router-dom';
-import { Card, Modal } from 'react-bootstrap';
+import { Card, Modal } from 'react-bootstrap'
```

Comment:
ESLint will flag it as an error without the ;

Reply:
Fixed it 👍🏻